model/game.js

```diff
@@ -11,4 +12,10 @@ Game.init({
   type: DataTypes.STRING
 }, { sequelize, modelName: 'game' });
 
+function associate() {
```

Comment:
Måten du gjer det her og i gameMode.js har to forskjellige effektar. I denne fila så vil `Game.associate =` bli returverdien av `associate()`. Men i gameMode.js så blir `GameMode.associate = ` satt til å peike på funksjonen din. 

Kan godt hende du får same resultat, men er lurt å vere konsekvent med kva versjon ein brukar